src/pages/debrief/debrief.html

```diff
@@ -19,6 +15,9 @@ <h4>Test outcome</h4>
       <h1>unsuccessful</h1>
     </div>
   </div>
+</ion-header>
```

Comment:
What would be the reason for moving the `Test Outcome` into the ion-header`?

Reply:
It's so that the content can scroll underneath. I spoke to Oli and he wants the list of faults to be in the background where you can scroll and then the test outcome and end debreif buttons be static in the foreground.

Reply:
Okay, that's cool.